lint.sh

```diff
@@ -78,17 +78,25 @@ check_bash(){
 }
 
 check_python(){
-  flake8 --exclude=.lintvenv . \
+  flake8 --exclude=.lintvenv,webhooktranslator . \
```

Comment:
Because it may contain build environments that don't pass flake8, also it handles its own flake8 via tox.